redash/query_runner/amazon_elasticsearch.py

```diff
@@ -0,0 +1,56 @@
+from .elasticsearch import ElasticSearch
+from . import register
+
+try:
+    from requests_aws4auth import AWS4Auth
+    enabled = True
+except ImportError:
+    enabled = False
+
+
+class AmazonElasticsearchService(ElasticSearch):
+    @classmethod
+    def name(cls):
+        return "Amazon Elasticsearch Service"
+
+    @classmethod
+    def enabled(cls):
+        return enabled
+
+    @classmethod
+    def type(cls):
+        return "aws_es"
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            'type': 'object',
+            'properties': {
+                'server': {
+                    'type': 'string',
+                    'title': 'Endpoint'
+                },
+                'region': {
+                    'type': 'string',
```

Comment:
No title here?

Reply:
No, because the default would be `Region`.